### ISSUE TITLE
Create metamask user-name registrar service

### ISSUE THREAD

**Issue by flyswatter** [MetaMask/metamask-plugin#1132]
We should grab a friendly ENS domain, and readily offer its subdomains to our users, to help coherent nicknames to become common.

Not posting our candidates here to avoid bid sniping.

**Comment by flyswatter** [comment]
Open questions regarding the requirements to register a name on our user registrar:
- Do we want any KYC steps to register names?
- Do we want to charge anything?  Maybe more for shorter names?  Do we care to do that, or are we happy enough to offer the service + get the name recognition?

**Comment by chafey** [comment]
hmm - wouldn't it be better to implement a more complete user profile service rather than using ENS?

**Comment by flyswatter** [comment]
Could do both. ENS is good for receiving, even cross client.

- Dan

**Comment by chafey** [comment]
Yes it could, but I like simplicity.  Not sure simply mapping an account to ENS is enough.  Are you aware of any kind of user profile initiative or smart contract?  I know there are plenty of existing schemas used in the internet to describe people, contacts, pictures, etc

**Comment by flyswatter** [comment]
It sounds like you're describing uPort, and we are already planning on an integration.

- Dan
